code/__HELPERS/_auxtools_api.dm

```diff
@@ -18,13 +26,13 @@ GLOBAL_LIST_EMPTY(auxtools_initialized)
 	}\
 
 #define AUXTOOLS_SHUTDOWN(LIB)\
-	if (GLOB.auxtools_initialized[LIB] == AUXTOOLS_FULL_INIT && fexists(LIB)){\
+	if (GLOB.auxtools_enabled && GLOB.auxtools_initialized[LIB] == AUXTOOLS_FULL_INIT && fexists(LIB)){\
```

Comment:
this actually kinda sucks (not the code, the edge case).

if auxtools is some how allowed to initialize, this has to run or world/reboot will crash the server.

but also on a prod server running tgs of any version, this doesn't matter because world/reboot crashing will just turn into a full restart.

Reply:
the biggest edge case i can see leading to this would be reloading the configuration using the verb in the debug tab.

Reply:
If you remove the GLOB.auxtools_enabled, this will function normally because GLOBs.auxtools_initialized only gets set when auxtools is initialized.